certified-connectors/Aranda Service Management/Readme.md

```diff
@@ -0,0 +1,327 @@
+## Aranda Service Management Suite (ASMS)
+
```

Comment:
Can you please update readme.md file as per the template link below?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1

Reply:
readme.md was fixed according to the attached documentation